src/components/YearSelection.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem
+} from 'reactstrap';
+
+const DropdownMenuStyle = {
+  height: '200px',
+  overflow: 'auto'
+};
+
+class YearSelection extends Component {
```

Comment:
Maybe naming this `YearSelector` would make more sense?

Reply:
Renamed to YearSelector